packages/css-library/src/stylesheets/core.scss

```diff
@@ -6,15 +6,22 @@
 
 //============ MIXINS & OTHER FUNCTIONS ==============//
 @use './override-function' as *;
-@use'./functions' as *;
+@use './functions' as *;
 @use './mixins' as *;
 
 //============ VENDOR PARTIALS ==============//
 @import 'lib/normalize';
 
+// Grid/columns classes
+@import "./formation-overrides/foundation-sites";
```

Comment:
This should handle being a bridge for the Foundation imports being [imported into Formation's core stylesheet here](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation/sass/core.scss#L9-L10) since these classes are still widely used in vets-website.